networkit/cpp/generators/test/ErdosRenyiEnumeratorGTest.cpp

```diff
@@ -54,9 +53,9 @@ static void testEre(const bool directed, const node n, const double prob) {
             ASSERT_FALSE(edge_exists[edgeindex(u, v, n)]);
             edge_exists[edgeindex(u, v, n)] = 1;
 
-            // collect statistics for probabilisitic checks
+            // collect statistics for probabilistic checks
             edge_counts[edgeindex(u, v, n)]++;
-            num_edges_thread[cacheline_scale * tid]++;
+            num_edges_thread[tid]++;
```

Comment:
Did this test fail before? How does this relate to the issue addressed by this PR?

Reply:
Yes, for example, see the current build on master. For MSVC builds, the previous calculation of std::count_if together with the `cacheline_scale` led to wrong results. The refactoring doesn't change the logic of the test, however the build now works.